### ISSUE TITLE
REST-API Link needed 

### ISSUE THREAD

**Issue by bel20git** [hzi-braunschweig/SORMAS-Project#5085]
I've communicated with a GA that is ready to go with a self developed Tool, that is able to use the SORMAS-REST-API.
The problem is at the moment, that they need the so called REST-API aink / adress the communication towards SORMAS is directed to.
A User with the Rest-User role in SORMAS is existant but probably has only this role, i've heard that could be the problem.
With the current configuration they receive an '401' error.

I would be happy to read some ideas how to solve this problem.

kind regards

**Comment by MateStrysewske** [comment]
@bel20git Yes, that's correct, right now that user needs additional roles in order to be able to view data. For a German Gesundheitsamt, I'd suggest the National User and/or Surveillance Supervisor role because that one has access to all data on the server.
